CHANGELOG.md

```diff
@@ -2,8 +2,17 @@
 
 ## Upcoming
 
+## v0.7.2 (2025-03-11)
+
+* Update the RabbitMQ library to v1.10.0.
+* Modify main.go to return help when no queue is provided
+* Add Dockerfile and .dockerignore.
+* Add scripts to build_container, test and wait for dependant test
```

Comment:
Please add credit to yourself (for example, see the first entry under v0.7) for these items.